apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+
+Lastly, the XM Cloud CLI allows you to manage your XM Cloud instance and deploy, instead of using the GUI. You can also serialize content items that represent developer artifacts from your XM cloud instance into your source code repository. To learn more about serialization with Sitecore, navigate [here](https://doc.sitecore.com/xmc/en/developers/xm-cloud/serialization-in-sitecore.html). 
+
+ Once the provisioning and  build process are finished, the deployment starts. The moment the deployment is finished, you can start using the new XM Cloud environment. Some actions run in the background to warm up the application..  
+
+**Congratulations!** You have successfully connected to XM Cloud and built your first project, that has an environment that is set up and ready to be used. The solution code is ready and connected to a CI/CD pipeline as well – these steps that can take days or weeks are completed within minutes here.  
+
+## Setup a Site Collection and Site  
+
+Now, that the deployment is finished, you will create your first site in XM Cloud.  
+
+1. You can access XM Cloud via the XM Cloud Deploy app through the “Go to XM Cloud” button 
+![XM Cloud Deploy Logs - Finished, Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/51576832a2594fde8dd0fb7f86fbdfd1?v=59c586b8) 
+If you closed that screen already you can also access XM Cloud from the Sitecore Cloud Portal. Click the “Open app” button on the top right. 
+![Sitecore Cloud Portal - Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/ef83f2ef7ff94688b9f84b59f57f98b5?v=8bed26f3) 
+2. Under the sites tab, click create site. 
+![Create new Site](https://sitecorecontenthub.stylelabs.cloud/api/public/content/08c2c56cb94c4ce8969c691a8b6ff4d3?v=c79d9844) 
+Here, you can choose to create an empty site, or a basic site that already has some content in it.  
+
+It is recommended to use the empty site as a starting point when you are creating your own websites outside the scope of this blog. The basic site is more for quick demo purposes, and would require excess cleanup when used for a client project.  However, for this blog, you will select to use the basic site.   
+![Choose Site Template](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1480ca22c0724a7496c9617b5f65cd6a?v=cf3753f9) 
+
+3. Name the site `Company DEV`.  
+![Name new Site and Site Collection](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1dc9460c15fd452681187485fae6fe37?v=8feccf45) 
+
+4. Select a pre-configured language, in this case, English.  
+
+At this time, you can also select an existing Site identifier, or create a new one. This is required when connecting CDP for embedded analytics and personalization.  
```

Comment:
Can we link out to a Sitecore article here on what CDP is? If I'm completely new dev to this vertical (CMS w/ Marketing) I may become distracted here as I don't know what that is.  Even if I don't click on the link, it shows empathy to newcomers to have that resource there.

Reply:
This has been resolved